src/libraries/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -118,14 +130,29 @@ public void DumpRuntimeInformationToConsole()
 
             if (osd.Contains("Linux"))
             {
+                void OutputHandler(object sender, DataReceivedEventArgs e)
+                {
+                    string trimmed = e.Data?.Trim();
+                    if (!string.IsNullOrEmpty(trimmed) && trimmed[0] != '#') // skip comments in files
+                    {
+                        Console.WriteLine(e.Data);
+                    }
+                }
+
                 // Dump several procfs files and /etc/os-release
-                foreach (string path in new string[] { "/proc/self/mountinfo", "/proc/self/cgroup", "/proc/self/limits", "/etc/os-release" })
+                foreach (string path in new string[] { "/proc/self/mountinfo", "/proc/self/cgroup", "/proc/self/limits", "/etc/os-release", "/etc/sysctl.conf", "/proc/meminfo" })
                 {
                     Console.WriteLine($"### CONTENTS OF \"{path}\":");
                     try
                     {
-                        using (Process cat = Process.Start("cat", path))
+                        using (Process cat = new Process())
                         {
+                            cat.StartInfo.FileName = "cat";
+                            cat.StartInfo.Arguments = path;
+                            cat.StartInfo.RedirectStandardOutput = true;
+                            cat.OutputDataReceived += OutputHandler;
```

Comment:
Nit: why use a local function rather than just definining this as a lambda?
```C#
cat.OutputDataReceived += (sender, e) =>
{
    if (e.Data != null && e.Data.Trim()[0] != '#')  // skip comments in files
    {
        Console.WriteLine(e.Data);
    }
};
```
?